src/fentoimage/board.py

```diff
@@ -4,12 +4,18 @@
 from chess import Board
 from PIL import Image, ImageDraw, ImageFont
 
-from fentoimage.config import Config
-from fentoimage.piece import PieceImage
```

Comment:
this change breaks on my machine:
```
Python 3.10.4 (main, Mar 31 2022, 08:41:55) [GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import chess
>>> from fentoimage.board import BoardImage
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ahmedk/Documents/Projects/python/fen-to-image/src/fentoimage/board.py", line 7, in <module>
    from config import Config
ModuleNotFoundError: No module named 'config'
```
When working on a Python library, you can install it in editable mode by going to the root project directory and typing `pip install -e .` in your terminal. This should fix your import issues by allowing you to import the library as you would outside of the project.